src/Sulu/Component/Security/Authorization/AccessControl/AccessControlManager.php

```diff
@@ -87,7 +87,30 @@ public function getUserPermissions(SecurityCondition $securityCondition, UserInt
         $checkPermissionType = empty($objectPermissions);
 
         $securityContextPermissions = $this->getUserSecurityContextPermissions(
-            $securityCondition,
```

Comment:
no not really because the security condition requires an object which i do not have ...